RobotCode/src/main/java/frc/robot/subsystems/ShooterMountSubsystem.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamSparkMAX;
+import frc.robot.constants.Ports;
+import java.util.Scanner;
+
+public class ShooterMountSubsystem extends SubsystemBase
+{
+
+	private TeamSparkMAX mainMotor, followMotor;
+
+	private double currentRotation;
+
+	private double originalRotation;
+	private double goalRotation;
+
+	private static final double CONVERSION_NUMBER = 180 / 21;
+
+	private static ShooterMountSubsystem instance;
+
+	public static ShooterMountSubsystem getInstance()
```

Comment:
We don't need this class to be a singleton, so remove this

Reply:
What the fuck did you just fucking post about me, you absolute beginner? I'll have you know I worked for ten of the biggest silicon-valley industry companies, and I've been involved in over two hundred top secret projects including NodeJS. I am trained in refactoring the most fucked up code, and I'm the top C++er in the entire fucking internet-connected universe. You are nothing to me, but just another IP. I will fucking revoke your commits from your gitlab account with absolute dedication using only one Rasperry Pi client. Mark my fucking words. You think you can get away with posting that shit on one of my numerous very personal blogs? Your devices are fucking bricked, kid. My attack software can be anywhere, anytime, and it is tasked to remove your entire git contributions from planet earth. Not only am I extensively trained in remote cross-firewall device-hacking, but I have access to over 100 of the United States CIA and NSA git repositories. If only you could have known what doom-bringing C-one-liner you have raised from my fucking hands, maybe you would have held your fingers. But you could not. You did not. And now you're paying the price, noob. I will hail havoc upon your puny online-presence and you will drown in your own badly designed software. You're fucking offline, kiddo.

Reply:
^ comment by yours truly, harrison
(not by mr charlie chaplin over here)